bench/db_bench.ml

```diff
@@ -7,6 +7,24 @@ let src = Logs.Src.create "db_bench"
 
 module Log = (val Logs.src_log src : Logs.LOG)
 
+let all = ref false
+
+and with_lmdb = ref false
+
+and with_index = ref false
+
+let speclist =
+  [
+    ("-all", Arg.Set all, "run all benchmarks");
```

Comment:
Can you use `--` instead of `-`?

Reply:
But this could be done in a separate PR using cmdliner

Reply:
I hesitated between Arg and Cmdliner, why do we prefer the latter?

Reply:
because it's generating a nice `--help` document too :-) (and we are using it in all the other MirageOS projects)